codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/RustModule.kt

```diff
@@ -25,6 +25,12 @@ data class RustModule(val name: String, val rustMetadata: RustMetadata, val docu
 
         val Config = public("config", documentation = "Configuration for the service.")
         val Error = public("error", documentation = "All error types that operations can return.")
-        val Operation = public("operation", documentation = "All operations that this crate can perform.")
+
+        /**
+         * Helper method to generate the `operation` Rust module.
+         * Its visibility depends on the generation context (client or server).
+         */
+        fun operationModule(visibility: Visibility): RustModule =
```

Comment:
I don't think you need `Module` in the name here (it's already used as `RustModule. ...`).

Reply:
I added the suffix to stress that it's not a "general purpose" constructor (like `public` or `private` or `default`), but it might have been too heavy handed. I'll get rid of it.